### PR TITLE
Make sure our production bundle is ES5 compatible.

### PR DESCRIPTION
**Pull request by luisrudge**
Fix https://github.com/auth0/auth0-spa-js/issues/97

### Description

Our production bundle was shipping ES2015 code, which is not supported in IE11. This PR fixes it and also adds a check to make sure our production build is ES5 compatible.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -8,13 +8,12 @@ import livereload from 'rollup-plugin-livereload';
...
 
 import pkg from './package.json';
-import { Object } from 'core-js';
 
 const EXPORT_NAME = 'createAuth0Client';
 
 const isProduction = process.env.NODE_ENV === 'production';
 const shouldGenerateStats = process.env.WITH_STATS === 'true';
-const getPlugins = (shouldMinify, tsconfigOverride = {}) => {
+const getPlugins = (shouldMinify, { target }) => {
```
Comment (damieng) [inline rollup.config.js]: Is this a breaking change for people who were providing additional options?

**Comment by echee** [comment]
will this also fix the IE11 issue with the way crypto is used in the production bundle?

**Comment by luisrudge** [comment]
@echee using msCrypto? If you load it via CDN, you can use the legacy bundle which has this included. If you have a bundler, you're probably patching this yourself, right?

**Comment by echee** [comment]
@luisrudge currently no, we are installing the npm package. We'd tried to polyfill this ourselves but thought surely it should be built in if the browser support is for IE11 too. Correct me if i'm wrong/

**Comment by luisrudge** [comment]
@echee My first thought was to let devs consuming the package via npm to do it themselves, but I think you're right! We should provide this OOTB. I'll patch this in the next release. Thanks for bringing this up!

**Pull request merged**